README.md

```diff
@@ -937,6 +938,19 @@ Table of Contents
   * [hashnode.com](https://hashnode.com/) — Hassle-free Blogging Software for Developers!.
   * [medium.com](https://medium.com/) — Get smarter about what matters to you.
 
+## Commenting Platforms
+  * [ReplyBox](https://getreplybox.com) - ReplyBox, a simple, honest comment system. No ads, no dodgy affiliate links, no fluff.
+  * [Staticman](https://staticman.net/) - Staticman is a Node.js application that receives user-generated content and uploads it as data files to a GitHub and/or GitLab repository, using Pull Requests.
+  * [GraphComment](https://graphcomment.com/) - GraphComment is a comments platform that helps you build an active community from website’s audience. 
+  * [Utterances](https://utteranc.es/) - A lightweight comments widget built on GitHub issues. Use GitHub issues for blog comments, wiki pages and more!
+  * [Talkyard](https://www.talkyard.io/blog-comments) - Talkyard is a commenting system for your website. Open source, install for free
+  * [Isso](https://posativ.org/isso) - A lightweight commenting server written in Python and JavaScript. It aims to be a drop-in replacement for Disqus.
+  * [Commento](https://commento.io/) - A fast, privacy-focused commenting platform
```

Comment:
This is free if you make significant open source contributions to their software, does not quality